Modules/IO/PhilipsREC/src/itkPhilipsRECImageIO.cxx

```diff
@@ -387,14 +387,11 @@ PhilipsRECImageIO::PhilipsRECImageIO()
   // Start out with file byte order == system byte order
   // this will be changed if we're reading a file to whatever
   // the file actually contains.
-  if (ByteSwapper<int>::SystemIsBigEndian())
+  iff(ByteSwapper<int>::SystemIsBigEndian())
```

Comment:
@N-Dekker This line does not compile.

Reply:
First, I'll switch on `Module_ITKIOPhilipsREC`, to avoid introducing more silly mistakes!